dask/tests/test_delayed.py

```diff
@@ -574,7 +574,7 @@ def test_finalize_name():
     def key(s):
         if isinstance(s, tuple):
             s = s[0]
-        return s.split("-")[0]
+        return s.split("-")[0].replace("_", "")
```

Comment:
Since `"ones_like"` will show up here and `"_"` is not an alphabetical character, replace `"_"` with `""` to ensure only alphabetical characters show up

Reply:
Included in a comment now as well